tests/helper/helper_generic.go

```diff
@@ -336,6 +336,21 @@ func GjsonMatcher(values []gjson.Result, expected []string) bool {
 	return matched == numVars
 }
 
+// GjsonExactMatcher validates if []results from gjson.GetMany match the expected values for each json path requested
+// Values is an array of results returned by the gjson.GetMany function
+// Expected is an array of strings, defines the expected values for each of the paths requested in the gjson.GetMany function
```

Comment:
Same question as in the suggestion for "values":
```suggestion
// "expected" is an array of strings, defines the expected values for each of the paths requested in the gjson.GetMany function
```

Reply:
idem